### PR TITLE
enhancement: Moved to fly.io!

### PR DESCRIPTION
**Pull request by Norbiros**

### CODE REVIEW

**Review by Sanielia — APPROVED** [review]
nice UwU

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# install psycopg2 dependencies.
+RUN apt-get update && apt-get install -y \
+    libpq-dev \
+    gcc \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p /code
```
Comment (goteusz-maszyk) [inline Dockerfile:14]: src instead of code would be better

**Review by goteusz-maszyk — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (Norbiros) [reply Dockerfile:14]: FlyCTL generated this file...

Comment (goteusz-maszyk) [reply Dockerfile:14]: idc

**Review by tapczanek1321 — APPROVED** [review]
idc about @goteusz-maszyk opinion so i approve ;33

**Pull request merged**